Automattic/simplenote-android
# Remove Large Dimensions

### Fix
Remove the `arrays.xml`, `dimens.xml`, and `integers.xml` files from the `values-large-xxxhdpi` and `values-xlarge-xxxhdpi` directories in order to have consistent text sizes across devices.

### Test
Since these changes target large and extra large devices with xxxhdpi density, use a device with those characteristics.
1. Open navigation drawer.
2. Notice text in navigation drawer is readable.
3. Close navigation drawer.
4. Notice text in note list is readable.
5. Tap any note in list.
6. Notice text in editor is readable.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
